freud/_order.pxd

```diff
@@ -70,20 +70,20 @@ cdef extern from "HexaticTranslational.h" namespace "freud::order":
 
 cdef extern from "Steinhardt.h" namespace "freud::order":
     cdef cppclass Steinhardt:
-        Steinhardt(unsigned int, bool, bool, bool, bool) except +
+        Steinhardt(vector[unsigned int], bool, bool, bool, bool) except +
         unsigned int getNP() const
         void compute(const freud._locality.NeighborList*,
                      const freud._locality.NeighborQuery*,
                      freud._locality.QueryArgs) except +
-        const freud.util.ManagedArray[float] &getQl() const
-        const freud.util.ManagedArray[float] &getQlm() const
-        const freud.util.ManagedArray[float] &getParticleOrder() const
-        float getOrder() const
+        const vector[freud.util.ManagedArray[float]] &getQl() const
+        const vector[freud.util.ManagedArray[float complex]] &getQlm() const
```

Comment:
This is making our CI sad. Is there a compatibility issue with that version of Cython, or something weird with this syntax?
```
Error compiling Cython file:
------------------------------------------------------------
...
        unsigned int getNP() const
        void compute(const freud._locality.NeighborList*,
                     const freud._locality.NeighborQuery*,
                     freud._locality.QueryArgs) except +
        const vector[freud.util.ManagedArray[float]] &getQl() const
        const vector[freud.util.ManagedArray[float complex]] &getQlm() const
                                                  ^
------------------------------------------------------------

/root/ci/freud/freud/_order.pxd:79:51: Expected ']', found 'complex'
```

Reply:
I am not sure, it passes on some tests, and locally, but not for some -_(-_-)_-? But I can typedef and see if that helps.